integration-tests/test_client_options.py

```diff
@@ -87,3 +88,26 @@ def test_support(insights_client):
     assert "Running command:" in support_result.stdout
     assert "Process output:" in support_result.stdout
     assert "Support information collected in" in support_result.stdout
+
+
+def test_check_show_results(insights_client):
+    """
+    Verify a remediation is returned when checking results from server (--check-results)
+    and then displaying then (--show-results)
+    First I will intorudce a vulnerability to the system and at the
+    end of the test I will set the correct chnod again
+    """
+    insights_client.register()
+    assert conftest.loop_until(lambda: insights_client.is_registered)
+
+    os.chmod("/etc/ssh/sshd_config", 0o777)
```

Comment:
Not sure what it is doing, however I don't see anything related to it in the output of `--check-results` when I run it locally on my own RHEL 9 VM

Reply:
we are introducing a vulnerability to the system to make sure that there will be an output using the --show-results - it doesn't really matter what vulnerability is being introduced  if the vulnerability is reversed at the end / which it does:)